x-pack/plugins/infra/public/components/asset_details/asset_details.stories.tsx

```diff
@@ -107,7 +101,7 @@ export const Page = PageTemplate.bind({});
 export const Flyout = FlyoutTemplate.bind({});
 Flyout.args = {
   renderMode: {
-    showInFlyout: true,
+    mode: 'flyout',
```

Comment:
I realized that this will conflict with my changes in [my PR](https://github.com/elastic/kibana/pull/163573) where I use `showInFlyout` as a boolean. I can adapt that once this PR is merged. It is all good, I am just curious now if it is somehow used in the telemetry object or not yet (I don't see the mode when I check it just want to be sure that I am not missing something)

Reply:
`renderMode` will only be used to determine the `componentName` attribute. I changed it because this way seems more explanatory to differentiate between render mode: `page` vs  `flyout`. Not sure if you agree.

Reply:
Ok, got it! Yeah, it makes sense, we can also extend with other modes that way! I used the boolean to have the "default" view and "flyout" view so the consumer should add a prop only in case a flyout is needed, but your change is good, thanks for explaining 👍